### PR TITLE
fix(postcss-merge-longhand): Don't explode declarations with custom properties

### LINKED ISSUE #540: Bizzare output on "border" values
Processing the following:
```css
#hovering-postform:checked ~ .post-form input[type=text],    
#hovering-postform:checked ~ .post-form input[type=password],
#hovering-postform:checked ~ .post-form textarea {           
  display: block;                                            
                                                             
  border: none;                                              
  margin: 0;                                                 
  border-bottom: var(--hovering-border);                     
  background-color: var(--hovering-bg);                      
  color: var(--hovering-fg);                                 
}                                                            
```
gives me the following.
```css
#hovering-postform:checked~.post-form input[type=password],#hovering-postform:checked~.post-form input[type=text],#hovering-postform:checked~.post-form textarea{background-col
or:#fff;background-color:var(--hovering-bg);border:none currentColor;border-bottom-width:var(--hovering-border);border-width:none none 1px;color:#000;color:var(--hovering-fg);disp
lay:block;margin:0}
```
Note things like `border:none currentColor;` and `border-bottom-width:var(--hovering-border);border-width:none none 1px;`.
This causes a weird border with the text color, while the unminified CSS doesn't have this issue.

PostCSS configuration:
```js
module.exports = {                 
  plugins: [                       
    require('postcss-preset-env')({
      stage: 3,                    
      browsers: 'defaults',        
    }),                            
    require('cssnano')({           
      preset: 'default',           
    }),                            
  ],                               
};                                 
```

### PR DESCRIPTION
**Pull request by andyjansson**
Fixes #540.

### CODE REVIEW

**Comment by equinusocio** [comment]
Yeah!

**Comment by equinusocio** [comment]
@evilebottnawi Can you made this fix and release an hotfix?

**Comment by evilebottnawi** [comment]
@equinusocio need exclude `yarn.lock` from this PR and i can merge and release patch

**Comment by equinusocio** [comment]
@evilebottnawi I will make a new PR in minutes.

File: packages/postcss-merge-longhand/src/lib/decl/boxBase.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import mergeValues from '../mergeValues';
 import remove from '../remove';
```
Comment (evilebottnawi) [inline packages/postcss-merge-longhand/src/lib/decl/boxBase.js]: https://github.com/cssnano/cssnano/pull/548/files#diff-62d2b7e64f1ece6f79b5c16fe2403b85R50

**Comment by evilebottnawi** [comment]
Close in favor https://github.com/cssnano/cssnano/pull/548

**Pull request closed without merging**